logging/performance_reliability/logging-content-filtering.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="logging-content-filtering"]
+= Filtering logs by content
+:context: logging-content-filtering
+
+toc::[]
+
+Collecting all logs from a cluster may produce a large amount of data, which can be expensive to transport and store.
+
+You can reduce the volume of your log data by filtering out low-value data that does not need to be stored. {logging} provides content filters that can be used to reduce the volume of log data. These filters use the same framework as the `kube-api-audit` filter.
```

Comment:
... consider dropping `These filters use the same framework as the `kube-api-audit` filter.`, I don't know that it is relevent